inversion_specifications/NAME-RHIME/overview.md

```diff
@@ -0,0 +1,57 @@
+# RHIME inverse model specification
+
+Transport model and inversion setup for the Univeristy of Bristol Regional Hierarchical Inverse Modelling Environment (RHIME). The following applies to all species, unless specified in a species specific file in this folder.
+
+## Transport model
+
+NAME model (version 7.2) developped by the UK Met Office (Muller et al. 2013), a Langrangian particle model that we run backwards in time for each measurement site used in the inversion.
+
+### Scope
+
+- Domain: 10.73°N to 79.06°N and from 97.9°W to 39.38°E.
+- Period: 2018-01-01 to 2023-01-01
+
+### Meteorology
+
+The meteorological fields from the UK Met Office Unified Model are used. The model is run on a grid of horizontal resolution 0.0135° x 0.0135° over the UK (Manning and Redington et al. 2021). Vertically, it uses 57 levels, expanding to 12.2km.
+
+### Model input parameters
+
+The langragian transport model is used with a release of 20000 particles/hour. Particles are released from the measurement altitude ± X m. Particles are tracked for 30 days prior to the measurement, or until they leave the domain.
```

Comment:
can you clarify how the particles are distributed vertically (and horizontally?)? I think they are released from a vertical line centred on the inlet, plus or minus some number of metres. But I may be wrong, and they could be released from a point. Can you check?

Reply:
Here, we also need to clarify what has been used to define the "surface" for the purposes of calculating the footprint. I **think** (but again, please check) that we assume that particles are considered to have interacted with the surface if they are within 40m of the ground